docs/building-modules.md

```diff
@@ -38,12 +38,140 @@ Hello universe!
 Zinnia provides all standard JavaScript APIs, you can find the full list in
 [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects).
 
-### console
+### Web APIs
+
+The following entities are defined in the global scope (`globalThis`).
+
+#### Console Standard
 
 Zinnia implements most of the `console` Web APIs like `console.log`. You can
 find the full list of supported methods in
 [Deno docs](https://deno.land/api@v1.30.3?s=Console) and more details about
 individual methods in
-[MDN web docs](https://developer.mozilla.org/en-US/docs/Web/API/console.)
+[MDN web docs](https://developer.mozilla.org/en-US/docs/Web/API/console)
+
+- [console](https://developer.mozilla.org/en-US/docs/Web/API/console)
+
+#### DOM Standard
+
+- [AbortController](https://developer.mozilla.org/en-US/docs/Web/API/AbortController)
+- [AbortSignal](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
+- [CustomEvent](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent)
+- [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event)
+- [EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget)
+
+#### Encoding Standard
+
+- [TextDecoder](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder)
+- [TextEncoder](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder)
+- [TextDecoderStream](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoderStream)
+- [TextEncoderStream](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoderStream)
+
+#### File API
+
+- [Blob](https://developer.mozilla.org/en-US/docs/Web/API/blob)
+- [File](https://developer.mozilla.org/en-US/docs/Web/API/File)
+- [FileReader](https://developer.mozilla.org/en-US/docs/Web/API/FileReader)
```

Comment:
I have mixed feelings about `Blob` and `File` related APIs. 

There is no access to the file system yet, so these APIs are only building blocks that cannot be meaningfully used (yet).

I also think the filesystem abstraction is not the right solution to allow modules to persist data. Something closer to a key/value store may be a much better fit. There are also LocalStorage and IndexedDB Web APIs (see [Deno's WebStorage API](https://deno.land/manual@v1.30.3/runtime/web_storage_api) and [MDN IndexedAPI](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API)) we may want to provide.

I am proposing to remove these three APIs for now. We can easily add them later, if there is a need.

WDYT?

Reply:
I removed these APIs in 8579744